tf/environments/dev/main.tf

```diff
@@ -269,6 +280,26 @@ module "ooniapi_cluster" {
   )
 }
 
+module "oonith_cluster" {
+  source = "../../modules/ecs_cluster"
+
+  name = "oonith-ecs-cluster"
+  key_name = module.adm_iam_roles.oonidevops_key_name
+  vpc_id = module.network.vpc_id
+  subnet_ids = module.network.vpc_subnet[*].id
+
+  asg_min = 2
+  asg_max = 6
+  asg_desired = 2
+
+  instance_type = "t2.small"
```

Comment:
We might want to tweak these a bit for prod, but for dev I think these are good values!